src/Status.h

```diff
@@ -123,7 +123,22 @@ typedef enum Status {
     /// example: it is outside allowed bounds).
     STATUS_INVALID_PARAMETER            = 30,
 
-    STATUS_MAX_VALUE                    = 30,
+    /// Indicates that client already received the result of the rpc.
+    /// It does not make sense to execute the RPC again. Most likely cause
+    /// is a delayed network packet.
+    STATUS_STALE_RPC                    = 31,
+
+    /// Indicates that the lease of a client is expired on the coordinator.
+    /// Master refused to execute the RPC with expired lease.
+    STATUS_EXPIRED_LEASE                = 32,
+
+    /// Indicates that a client tried to perform transaction operations after
+    /// the transaction commit had already started.
+    STATUS_TX_OP_AFTER_COMMIT           = 33,
+    STATUS_MAX_VALUE                    = 33,
+
+    //TODO(seojin): figure out why compile fails without this..
```

Comment:
Just a reminder to fix this (there shouldn't be a status with value higher than STATUS_MAX_VALUE)